WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -0,0 +1,618 @@
+        let page = pageObject as! Page
+        
+        cell.configureCell(page)
+    }
+    
+    private func cellIdentifierForPage(page: Page) -> String {
+        var identifier : String
+        
+        if recentlyTrashedPostObjectIDs?.containsObject(page.objectID) == true && currentPostListFilter()?.filterType != .Trashed {
+            identifier = self.dynamicType.restorePageCellIdentifier
+        } else {
+            identifier = self.dynamicType.pageCellIdentifier
+        }
+        
+        return identifier
+    }
+    
+    // MARK: - Post Actions
+    
+    func createPost() {
+        let navController : UINavigationController
+        
+        if EditPageViewController.isNewEditorEnabled() {
+            let postViewController = EditPageViewController(draftForBlog: blog)
+            
+            navController = UINavigationController(rootViewController: postViewController)
+            navController.restorationIdentifier = WPEditorNavigationRestorationID
+            navController.restorationClass = EditPageViewController.self
+        } else {
+            let editPostViewController = WPLegacyEditPageViewController(draftForLastUsedBlog: ())
+            
+            navController = UINavigationController(rootViewController: editPostViewController)
+            navController.restorationIdentifier = WPLegacyEditorNavigationRestorationID
+            navController.restorationClass = WPLegacyEditPageViewController.self
+        }
+        
+        navController.modalPresentationStyle = .FullScreen
+        
+        presentViewController(navController, animated: true, completion: nil)
+        
+        WPAppAnalytics.track(.EditorCreatedPost, withProperties: ["tap_source": "posts_view"], withBlog: blog)
+    }
+    
+    private func editPage(apost: AbstractPost) {
+        WPAnalytics.track(.PostListEditAction, withProperties: propertiesForAnalytics())
+        
+        if EditPageViewController.isNewEditorEnabled() {
+            let pageViewController = EditPageViewController(post: apost, mode: kWPPostViewControllerModePreview)
+            
+            navigationController?.pushViewController(pageViewController, animated: true)
+        } else {
+            // In legacy mode, view means edit
+            let editPageViewController = WPLegacyEditPageViewController(post: apost)
+            let navController = UINavigationController(rootViewController: editPageViewController)
+            
+            navController.modalPresentationStyle = .FullScreen
+            navController.restorationIdentifier = WPLegacyEditorNavigationRestorationID
+            navController.restorationClass = WPLegacyEditPageViewController.self
+            
+            presentViewController(navController, animated: true, completion: nil)
+        }
+    }
+    
+    private func draftPage(apost: AbstractPost) {
+        WPAnalytics.track(.PostListDraftAction, withProperties: propertiesForAnalytics())
```

Comment:
Created [a ticket](https://github.com/wordpress-mobile/WordPressCom-Analytics-iOS/issues/66) to handle this.